src/basic-syntax/slices.md

```diff
@@ -14,3 +14,17 @@ fn main() {
 
 * Slices borrow data from the sliced type.
 * Question: What happens if you modify `a[3]`?
+
+<details>
+
+* We create a slice by borrowing `a` and specifying the starting and ending indexes in brackets.
+
+* If the slice starts at index 0, Rust’s range syntax means we can drop the starting index. 
+    
+* The same is true for the last index, so `let s: &[i32] = &a[2..len];` or `let s: &[i32] = &a[2..];` are equal.
+
+* We set `s` as a reference of `i32`s. Notice that the type of `s` no longer has an array length. This avoids type errors when performing computations on slices of different sizes.
```

Comment:
```suggestion
* `s` is a reference to a slice of `i32`s. Notice that the type of `s` no longer mentions the array length. This allows us to performing computations on slices of different sizes.
```

Reply:
Thanks Martin, we just need 'perform' vs 'performing'...is it easier if you adjust the change?